src/Event/Event.php

```diff
@@ -188,7 +188,7 @@ public function setResult($value = null)
      * Access the event data/payload.
      *
      * @param string|null $key The data payload element to return, or null to return all data.
-     * @return array|null The data payload if $key is null, or the data value for the given $key. If the $key does not
+     * @return mixed[]|mixed|null The data payload if $key is null, or the data value for the given $key. If the $key does not
```

Comment:
How is different `mixed[]` from `array`? Also, `array|mixed|null` doesn't make sense, as `mixed` contains `array` and `null`.